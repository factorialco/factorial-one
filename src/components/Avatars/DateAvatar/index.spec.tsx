import { render } from "@testing-library/react-native";
import React from "react";
import { DateAvatar } from "./";

describe("DateAvatar", () => {
  it("Snapshot", () => {
    const fixedDate = new Date("2025-01-01T00:00:00Z");
    const { toJSON } = render(<DateAvatar date={fixedDate} />);

    expect(toJSON()).toMatchSnapshot();
  });
});
