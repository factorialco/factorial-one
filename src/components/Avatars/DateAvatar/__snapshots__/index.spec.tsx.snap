// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`DateAvatar Snapshot 1`] = `
<View
  className="flex h-10 w-10 flex-col items-center justify-center rounded border border-solid border-f1-border-secondary bg-f1-background-inverse-secondary"
>
  <Text
    className="pt-0.5 text-xs font-semibold uppercase leading-3 text-f1-special-highlight dark:text-f1-foreground-inverse-secondary"
  >
    Jan
  </Text>
  <Text
    className="flex items-center justify-center text-lg font-medium leading-tight text-f1-foreground"
  >
    1
  </Text>
</View>
`;
